### PR TITLE
nbd: backup and recover curvetab

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: nbd/nbd-package/DEBIAN/preinst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+CURVETAB=/etc/curve/curvetab
+CURVETABBAK=/etc/curve/curvetab-bak
+
+if [ -f ${CURVETAB} ]; then
+    cp -f ${CURVETAB} ${CURVETABBAK}
```
Comment (SeanHai) [inline nbd/nbd-package/DEBIAN/preinst:9]: Add '-p' cp command option to preserve 'mode,ownership,timestamps' ?

File: nbd/nbd-package/DEBIAN/postinst [hunk 2]
```diff
@@ -6,4 +6,12 @@ chmod +x /usr/bin/map_curve_disk.sh
 cp /usr/bin/map_curve_disk.sh /etc/init.d
 update-rc.d map_curve_disk.sh defaults
 
+CURVETAB=/etc/curve/curvetab
+CURVETABBAK=/etc/curve/curvetab-bak
+
+if [ -f ${CURVETABBAK} ]; then
+    cp -f ${CURVETABBAK} ${CURVETAB}
```
Comment (SeanHai) [inline nbd/nbd-package/DEBIAN/postinst:13]: Add '-p' cp command option to preserve 'mode,ownership,timestamps' ?

File: nbd/nbd-package/DEBIAN/preinst (hunk 1 above)
Comment (ilixiaocui) [reply nbd/nbd-package/DEBIAN/preinst:9]: Don’t you need to copy the contents of the existed curvetab ?

Comment (wu-hanqing) [reply nbd/nbd-package/DEBIAN/preinst]: fix

File: nbd/nbd-package/DEBIAN/postinst (hunk 2 above)
Comment (wu-hanqing) [reply nbd/nbd-package/DEBIAN/postinst]: fix

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**